docs/running_rune_with_occlum.md

```diff
@@ -7,7 +7,8 @@
 - Assume the host system is CentOS 7.5.
 - You can also launch a CentOS 7.5 container as your host system with the following command:
   ```shell
-
+  docker pull centos:centos7.5.1804
```

Comment:
docker pull can be skipped. Docker run already does the pull operation.

Reply:
ok